modules/SocketManager/module/src/socketmanager.c

```diff
@@ -370,6 +370,11 @@ ind_soc_socket_unregister(int socket_id)
 
 static ind_soc_run_status_t ind_soc_run_status__;
 
+/*
+ * NOTE: using this api to exit ind_soc_select_and_run may not always result in
+ * a timely exit if the registered callbacks are long. Consider using other
```

Comment:
to what does "registered callbacks are long" refer?
do you mean timer events with large values for repeat_time_ms?  or sockets that infrequently are read ready?

Reply:
*registered callbacks have long timer durations.

Reply:
suggest rewording "registered callbacks are long" to "timer events with long times between timer callbacks"

Reply:
sure will do.